.github/workflows/build_deploy_latest.yml

```diff
@@ -11,11 +11,12 @@ jobs:
     strategy:
       matrix:
         config:
-        - { image_name: ubuntu1604-build, dockerfile: vowpal_wabbit/ubuntu1604-build.Dockerfile }
-        - { image_name: ubuntu1404-build, dockerfile: vowpal_wabbit/ubuntu1404-build.Dockerfile }
-        - { image_name: centos7_6_1810-build, dockerfile: vowpal_wabbit/centos7_6_1810-build.Dockerfile }
-        - { image_name: rl-ubuntu-1604, dockerfile: reinforcement_learning/ubuntu1604-build.Dockerfile}
-        - { image_name: all-dev-ubuntu1604, dockerfile: development/ubuntu1604-dev.Dockerfile }
+        - { image_name: ubuntu1604-build, dockerfile: vowpal_wabbit/ubuntu1604-build.Dockerfile, context: ./ }
```

Comment:
nit: I'm guessing the default context is needed and there's no other way to default to this in all the cases except manylinux?

Reply:
I was trying to work this out - there's sneaky things we might be able to do but I just opted for the simple solution

Reply:
yea, thats what i suspected